packages/quizzes-dashboard/development.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:11-alpine
+
+USER node
+VOLUME /quizzes/dashboard
+WORKDIR /quizzes/dashboard
+
+EXPOSE 5000
+ENTRYPOINT ["/bin/sh", "-c"]
+CMD [ "npm ci && npm run build && npm run dev" ]
```

Comment:
Can we `npm ci` and `npm run build` during the build with two different run commands? That would cache the operations and we wouldn't have to run these again each time we start the container.